benedict/dicts/keylist/keylist_dict.py

```diff
@@ -28,7 +28,10 @@ def __delitem__(self, key):
 
     def _delitem_by_keys(self, keys):
         parent, key, _ = keylist_util.get_item(self, keys)
-        if type_util.is_dict_or_list(parent):
+        if type_util.is_wildcard(key):
+            self[keys[:-1]] = []
```

Comment:
Maybe it's better to empty the existing list `list.clear()` without creating a new one, in this way if the list is referenced somewhere, the pointer doesn't change. Do you agree?

Reply:
yep.

Reply:
Sometimes I forget parts of base API.